react-components/src/components/Reveal3DResources/Reveal3DResources.tsx

```diff
@@ -62,6 +69,11 @@ export const Reveal3DResources = ({
     [reveal3DModels]
   );
 
+  const { data: assetMappings } = useAssetMappedNodesForRevisions(cadModelOptions);
+
+  void useGenerateAssetMappingCachePerItemFromModelCache(cadModelOptions, assetMappings);
+  void useGenerateNode3DCache(cadModelOptions, assetMappings);
```

Comment:
@pramodcog did some work recently where we fetch all asset mappings when the scene is loaded, so that we can eventually color the contextualized nodes. I assume there's some overlap between the data you fetch here, and the data we're fetching for that. Could we make sure that those two data fetching workloads are happening together?

Reply:
Yeah.. I'm gonna check with him

Reply:
`useAssetMappedNodesForRevisions` is a cached mappings when scene is loaded, so it is re-used is several places, which I think is used the right way here.